### PR TITLE
More polish to visual report(pdf and png)

### PR DESCRIPTION
**Pull request by zhongnansu**
*Issue #, if available:*

*Description of changes:*
- remove all buttons
- change the setContent to use template with more css stylesheet
- TODO: the text in pdf is still not selectable, will probably add in the next phase as enhancement

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: kibana-reports/server/routes/utils/report_template.html [hunk 1]
```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>%REPORT NAME%</title>
```
Comment (kgcreative) [inline kibana-reports/server/routes/utils/report_template.html:6]: Let's either add a generic title here, or insert the report title, if we're able to get it
```suggestion
    <title>Kibana Reports</title>
```

**Review by kgcreative — COMMENTED** [review]
LGTM, pending the comment re: page title.

**Review by davidcui1225 — APPROVED** [review]
LGTM

**Review by joshuali925 — APPROVED** [review]
LGTM

**Comment by iget-master** [comment]
@zhongnansu please see #350

**Pull request merged**